changelog.txt

```diff
@@ -1,4 +1,5 @@
 0.83
+ * Support async queries & transactions.
```

Comment:
"Support added for async queries and transactions"
Also this should be moved below the BREAKING changes.